pkg/assume/assume.go

```diff
@@ -221,6 +221,7 @@ func AssumeCommand(c *cli.Context) error {
 			if len(profileKeys) == 0 {
 				return clierr.New("Granted couldn't find any AWS profiles in your config file or your credentials file",
 					clierr.Info("You can add profiles to your AWS config by following our guide: "),
+					// TODO: this is a broken link, where should it point to?
 					clierr.Info("https://granted.dev/awsconfig"),
```

Comment:
I tried to track this down to see where it originally pointed to but wasn't able to find it. I think we previously had a redirect in CloudFlare to our docs page. Here's the best link:

```suggestion
					clierr.Info("https://docs.commonfate.io/granted/getting-started#set-up-your-aws-profile-file"),
```